### PR TITLE
Get "." and ".." back

### PR DESCRIPTION
**Pull request by rfjakob**
Also adds a test for them.

The change passed an xfstests run.

Fixes issue #62

### CODE REVIEW

File: fuse/nodefs/dir.go [hunk 1]
```diff
@@ -70,10 +70,6 @@ func (d *connectorDir) ReadDirPlus(input *fuse.ReadIn, out *fuse.DirEntryList) (
 			continue
 		}
 
-		if e.Name == "." || e.Name == ".." {
-			continue
-		}
-
```
Comment (hanwen) [inline fuse/nodefs/dir.go]: you should skip the lookup for . and .. below

Comment (rfjakob) [inline fuse/nodefs/dir.go]: Shouldn't readdirplus also return the extra infos for "." and ".."?

**Comment by rfjakob** [comment]
Ok, the kernel does not seem to ever stat "." and "..", instead it stats the (parent) dir directly. Fixed.

**Comment by rfjakob** [comment]
Already merged, thanks! Closing the pull request.

**Pull request closed without merging**